bin/hipcc

```diff
@@ -114,7 +114,7 @@ if ($HIP_PLATFORM eq "clang") {
     }
     $ROCM_PATH=$ENV{'ROCM_PATH'} // "/opt/rocm";
     $HIPCC="$HIP_CLANG_PATH/clang++";
-    $HIPCXXFLAGS .= "-std=c++11 -I$HIP_PATH/include";
+    $HIPCXXFLAGS .= "-std=c++11 -isystem $HIP_CLANG_PATH/../lib/clang/7.0.0/include -I$HIP_PATH/include";
```

Comment:
Does this need to be hard-coded to clang 7.0? Is there some way to get this path from some clang config/binary?

Reply:
Fixed. Thanks.